src/lxc/attach.c

```diff
@@ -853,11 +853,24 @@ static int attach_child_main(struct attach_clone_payload *payload)
 
 	if (init_ctx->container && init_ctx->container->lxc_conf &&
 	    init_ctx->container->lxc_conf->seccomp) {
-		ret = lxc_seccomp_load(init_ctx->container->lxc_conf);
+		struct lxc_conf *conf = init_ctx->container->lxc_conf;
+
+		ret = lxc_seccomp_load(conf);
```

Comment:
Just a heads up that this API has changed, you have to do a subsequent seccomp_fd_get(), it doesn't return directly from seccomp_load() any more.

Reply:
Oh yeah, I'm already doing that though. This is `lxc_seccomp_load()` not `seccomp_load()` and. The former calls `seccomp_fd_get()`. :)

Reply:
Oh, I thought I'd looked for it and didn't see it...